storage/innobase/handler/ha_innodb.cc

```diff
@@ -171,6 +171,8 @@ static const long AUTOINC_OLD_STYLE_LOCKING = 0;
 static const long AUTOINC_NEW_STYLE_LOCKING = 1;
 static const long AUTOINC_NO_LOCKING = 2;
 
+static const ulong srv_buf_pool_chunk_min_size = 1024 * 1024;
```

Comment:
Playing @vaintroub’s advocate: Because Microsoft uses the LLP64 data model and everyone else the LP64 data model, it is better to avoid using the `ulong` or `long` data types at all. You probably meant something like this:
```c++
static constexpr size_t buf_pool_chunk_min_size= 1U << 20;
```
I think that this could be defined as a public member somewhere inside `buf_pool_t`. I would not use the `srv_` prefix in new code. I never was a fan of obfuscating configuration parameters in the code by using a different prefix.

Reply:
👏